.github/actions/configure-environment/action.yml

```diff
@@ -13,11 +13,38 @@ runs:
         /#.*arch=/ && $0 ~ "arch="arch { print }
         ' environment.yml > ci-environment.yml
 
-    - uses: conda-incubator/setup-miniconda@v2
+    - name: Setup Mambaforge
+      uses: conda-incubator/setup-miniconda@v2
       with:
+        miniforge-variant: Mambaforge
+        miniforge-version: latest
+        # Do not specify environment file - see Cache step below
```

Comment:
So at this point, we have an empty environment?

Reply:
Yes, the env always be updated in the last step, whether the cache exists or not.